Glyssen/Bundle/GlyssenDblTextMetadata.cs

```diff
@@ -84,6 +85,9 @@ public override GlyssenDblMetadataLanguage Language
 		[XmlElement("projectStatus")]
 		public ProjectStatus ProjectStatus = new ProjectStatus();
 
+		[XmlElement("projectsettings")]
```

Comment:
I realize that the immediately preceding element does not conform to this, but I think we typically use an initial capital letter for elements and an initial lowercase letter for attributes.